src/components/WYSIWYG/WYSIWYG.tsx

```diff
@@ -1,18 +1,23 @@
-import React, { FunctionComponent } from 'react';
+import { throttle } from 'lodash-es';
+import React from 'react';
+import 'trumbowyg/dist/trumbowyg.min';
 
-import Trumbowyg from 'react-trumbowyg';
+import 'trumbowyg/dist/langs/nl.min.js';
```

Comment:
declare module

Reply:
hoe bedoel je?

Reply:
Als deze geen types heeft toevoegen aan react-app-env.d.ts `declare module 'trumbowyg/dist/trumbowyg.min';`
Dan klaag typescript daar niet meer over in de editor.

Reply:
probeer uw tsc/docker eens te herstarten want bij mij klaagt hij daar niet over. Enkel als je `import * as test from 'test'` klaagt mijn tsc, maar dit is enkel een file importen.

![image](https://user-images.githubusercontent.com/1710840/72735671-94773a00-3b9c-11ea-8121-d756dc11a9f5.png)

Reply:
Ik had het gwn over de editor, daarmee dat we die ook zijn gaan beginnen toevoegen aan die file dacht ik:

![image](https://user-images.githubusercontent.com/16118473/72737528-0a30d500-3ba0-11ea-9133-a467ada56567.png)
Het gaat idd de linter of compiler van ts niet triggeren

Reply:
ah, ok, die hints zie ik niet in intellij.
Fixed